src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/ObjectWriter/R2RPEBuilder.cs

```diff
@@ -270,16 +260,17 @@ public int GetSymbolFilePosition(ISymbolNode symbol)
         /// Emit built sections into the R2R PE file.
         /// </summary>
         /// <param name="outputStream">Output stream for the final R2R PE file</param>
-        public void Write(Stream outputStream)
+        public void Write(Stream outputStream, int timeDateStamp)
```

Comment:
>int [](start = 47, length = 3)

This should be nullable or something to indicate that we don't want to force the timestamp to be reset here.

Reply:
Well, I think it's more about getting the proper <code>timeDateStamp</code> in the first place, we need to write something into the PE header.